src/apis/loadDocument.js

```diff
@@ -10,6 +10,7 @@
  * @param {boolean} [options.withCredentials] Whether or not cross-site requests should be made using credentials.
  * @param {string} [options.cacheKey] A key that will be used for caching the document on WebViewer Server.
  * @param {string} [options.password] A string that will be used to as the password to load a password protected document.
+ * @param {function} [options.decrypt] A custom function to decrypt the document
```

Comment:
Should I also document the decryptOptions?

Reply:
Yeah, let's document `decryptOptions` as well.
For the documentation, I think we can just borrow from `createDocument`.

```jsdoc
 * @param {boolean} [options.xodOptions.decrypt] - Function to be called to decrypt a part of the XOD file. For default XOD AES encryption pass CoreControls.Encryption.decrypt.
 * @param {boolean} [options.xodOptions.decryptOptions] -  An object with options for the decryption e.g. {p: "pass", type: "aes"} where is p is the password.
```

I guess the main part here is that we want to mention this is for XOD files only.

Reply:
Yes let's add it. It should be an object with properties `type` for the type of encryption and `p` for the password.

Reply:
yeah oops, the documentation for createDocument doesn't seem to be correct.